velox/functions/prestosql/tests/ArgGeneratorTest.cpp

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/expression/fuzzer/tests/ArgGeneratorTestUtils.h"
+#include "velox/functions/prestosql/fuzzer/DivideArgGenerator.h"
+#include "velox/functions/prestosql/fuzzer/FloorAndRoundArgGenerator.h"
+#include "velox/functions/prestosql/fuzzer/MultiplyArgGenerator.h"
+#include "velox/functions/prestosql/fuzzer/PlusMinusArgGenerator.h"
+#include "velox/functions/prestosql/tests/utils/FunctionBaseTest.h"
+
+using namespace facebook::velox;
+using namespace facebook::velox::fuzzer::test;
+namespace {
+
+class ArgGeneratorTest : public functions::test::FunctionBaseTest {
+ protected:
+  const FunctionSignatureMap allSignatures_ = getFunctionSignatures();
+};
+
+TEST_F(ArgGeneratorTest, plus) {
+  const auto signatures = getSignatures(allSignatures_, "plus", "decimal");
+  VELOX_CHECK_EQ(signatures.size(), 1);
+  const auto& signature = *signatures[0];
```

Comment:
This code repeats multiple time. Consider adding helper method:

```
const auto& signature = getOnlySignature("plus");
```

Reply:
Updated. Thanks.